<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(181, 181, 181);</string>
  </property>
  <widget class="QComboBox" name="cmb_musteri">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_sozlesme_kodu">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_baslangic">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_bitis">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_durum">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>AKTİF  /  PASİF</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_musteri">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Müşteri Adı</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_sozlesme_kodu">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Sözleşme Kodu</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_baslangic">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Sözleşme Baş. Tarihi</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_bitis">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Sözleşme Bitiş Tarihi</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_sozlesme_tipi">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_uzama_var">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>98</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Uzatma Var</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_uzama_yok">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>50</y>
     <width>98</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Uzatma Yok</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_arac_adedi">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>80</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_esnek_sefer">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>110</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Esnek Sefer</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_ucret_tipi">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_toplam_tutar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_kdv_orani">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_kdv_dahil_tutar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>KDV DAHİL TUTAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_kaydet">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>KAYDET</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_temizle">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>300</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>TEMİZLE</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbl_sozlesmeler">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>821</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_ucret_tipi">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Ücret Tipi</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_toplam_tutar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Toplam Tutar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_kdv_orani">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Kdv Oranı %</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_sozlesme_tipi">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Sözleşme Tipi</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_arac_adedi">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Araç Adedi</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_uzama">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Sözleşme Uzatma</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbl_fiyat_matrisi">
   <property name="geometry">
    <rect>
     <x>455</x>
     <y>150</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="btn_fiyat_ekle">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>150</y>
     <width>40</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListAdd"/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_fiyat_sil">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>180</y>
     <width>40</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListRemove"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
